test/helpers/node.go

```diff
@@ -334,10 +334,16 @@ func (s *SSHMeta) RenderTemplateToFile(filename string, tmplt string, perm os.Fi
 		return err
 	}
 
-	cmd := fmt.Sprintf("install -m %o <(echo '%s') %s\n", perm, content, filepath.Join(s.basePath, filename))
+	dst := filepath.Join(s.basePath, filename)
+	cmd := fmt.Sprintf("echo '%s' > %s", content, dst)
 	res := s.Exec(cmd)
 	if !res.WasSuccessful() {
 		return fmt.Errorf("%s", res.CombineOutput())
 	}
+	cmd = fmt.Sprintf("chmod %o %s", perm, dst)
+	res = s.Exec(cmd)
+	if !res.WasSuccessful() {
+		return fmt.Errorf("%s", res.CombineOutput())
```

Comment:
```suggestion
		return fmt.Error(res.CombineOutput())
```

Reply:
```
helpers/node.go:341:10: undefined: fmt.Error
helpers/node.go:346:10: undefined: fmt.Error
```

```
helpers/node.go:341:38: cannot use res.CombineOutput() (type *bytes.Buffer) as type string in argument to fmt.Errorf
helpers/node.go:346:38: cannot use res.CombineOutput() (type *bytes.Buffer) as type string in argument to fmt.Errorf
```
I'm assuming that is why the code was formatted this way.

Reply:
Yeah. Ok. Maybe. But... Ok.